utils/data/demos.js

```diff
@@ -141,6 +141,18 @@ const DEMOS = [
       twitter: 'MaxProgramming1',
       email: 'maxprogramming05@gmail.com',
     },
+  },
+  {
+    id: '_geolocation_',
+    title: 'Geolocation API Demo',
```

Comment:
Would be great to add a relevant emoji in the title like the one we have for others?

https://emojipedia.org/

Reply:
Okay, it wasn't mentioned in the requirements. Do you have any suggestions for which emoji you want to add ?

Reply:
U can use this one: https://emojipedia.org/anchor/